### PR TITLE
etc: update grafana config

### PR DESCRIPTION
**Pull request by nolouch**
@iamxy @GregoryIan  PTAL

### CODE REVIEW

File: op-guide/monitoring-tidb.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ Add the Push Gateway address to the yaml configuration file:
```
Comment (iamxy) [inline op-guide/monitoring-tidb.md]: please remove these `/r`

**Comment by iamxy** [comment]
LGTM

**Comment by iamxy** [comment]
don't forget to update docs-cn.

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**